scss/reset/_bodyText.scss

```diff
@@ -13,9 +13,12 @@ category: Text Styles
 body {
 	color: $C_textPrimary;
 	font-family: $font;
-	font-size: $font-size;
 	font-weight: $W_normal;
 	line-height: $line-height;
+
+	@include responsiveVarContext--scalingType() {
+		font-size: $scalingType-body;
```

Comment:
This is a major change, but it may be worth it; if all type scales at the same rate, we could use `rem` instead:

`body { font-size: $font-size; @include atMediaUp(medium) { font-size: $font-size * scaleRatio; } ... }`

`%text--big { font-size: rem($font-size--big); }`